Framework/PythonInterface/plugins/functions/TeixeiraWaterIqt.py

```diff
@@ -0,0 +1,77 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2007 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
Just a quick query before I gatekeeper this - should the copyright year on this file and the new test file for the function be 2024?

Reply:
Ahh.. you are right ! Thanks for noticing it. It is updated now.